omniport/core/kernel/utils/logs.py

```diff
@@ -0,0 +1,19 @@
+import logging
+
+logger = logging.getLogger('kernel_permissions')
+
+
+def log_permission(permission_name, user, is_allowed):
+    """
+    A common utility function to log requests for kernel permissions
```

Comment:
Function documentation should be imperative like
- Do X
- Change Y
- Return Z

Reply:
Done, please check.

Reply:
@ayushjn20 I don't see the difference.
```suggestion
    Form sentences to log requests for kernel permissions
```